frontend/src/metabase/visualizations/visualizations/SmartScalar/utils.unit.spec.ts

```diff
@@ -43,8 +43,16 @@ const getAutoPrecisionOptions = (width: number) => {
 describe("SmartScalar > utils", () => {
   describe("scalar.comparisons", () => {
     const FIELD_NAME = "Count";
-    const createInsights = (dateUnit: RelativeDatetimeUnit) => [
-      { unit: dateUnit, col: FIELD_NAME },
+    const createInsights = (dateUnit: DateTimeAbsoluteUnit): Insight[] => [
+      {
```

Comment:
It would be nice to create `createMockInsight` as we do with other entities

Reply:
Makes sense I can do that in another small PR so I can merge this right now